tools/olm-bundle/generate-olm-bundle.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+#
+# Generates OLM bundle using existing CRDs
+#
+#
+source ../multi-platform-support.sh
```

Comment:
I think this should look like this so that it can be run from different directories (e.g. `./tools/olm-bundle/generate-olm-bundle.sh ...`)

```suggestion
source "$(dirname $(realpath $0))/../multi-platform-support.sh"
```